src/pages/dar_application/ResearcherInfo.jsx

```diff
@@ -0,0 +1,394 @@
+                              disabled={!isNil(darCode) || readOnlyMode}
+                              ariaLevel={ariaLevel + 3}
+                              onChange={({ key, value }) => formFieldChange({ key, value })}
+                              validation={validation.cloudProviderType}
+                              onValidationChange={onValidationChange}
+                            />
+                          </div>
+                        </div>
+                      )
+                    }
+                    {
+                      formData.cloudUse === true && (
+                        <div className='row no-margin'>
+                          <div className='col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'>
+                            <FormField
+                              id='cloudProviderDescription'
+                              type={FormFieldTypes.TEXTAREA}
+                              defaultValue={formData.cloudProviderDescription}
+                              disabled={!isNil(darCode) || readOnlyMode}
+                              validators={[FormValidators.REQUIRED]}
+                              placeholder={'Please describe the type(s) of cloud computing service(s) you wish to obtain (e.g PaaS, SaaS, IaaS, DaaS) and how you plan to use it (them) to carry out the work described in your Research Use Statement (e.g. datasets to be included, process for data transfer) analysis, storage, and tools and/or software to be used. Please limit your statement to 2000 characters'}
+                              rows={6}
+                              maxLength={2000}
+                              ariaLevel={ariaLevel + 3}
+                              onChange={({ key, value }) => formFieldChange({ key, value })}
+                              validation={validation.cloudProviderDescription}
+                              onValidationChange={onValidationChange}
+                            />
+                          </div>
+                        </div>
+                      )
+                    }
+                  </div>
+                )
+              }
+            </div>
+          </div>
+        </div>
+
+        <div className='dar-application-row' data-cy='external-collaborators'>
+          <h3>1.9 External Collaborators</h3>
+          {includeInstructions && (
+            <div>
+              {'Please list External collaborators here. External Collaborators are individuals who are not employees of the Requesting PI\'s institution or do not work at the same location as the PI. They must be independently approved to access controlled-access data subject to the GDS Policy. While not required to have a Library Card, it is encouraged. External Collaborators must submit an independent DAR approved by their signing Official to collaborate on this project. They can add their Lab Staff via their DAR. Approval of this DAR does not indicate approval of the External Collaborators listed.'}
```

Comment:
See previous comment, I think this can be better expressed as plain text with html entitites